cobc/cobc.c

```diff
@@ -3131,6 +3139,20 @@ process_command_line (const int argc, char **argv)
 			cobc_wants_debug = 1;
 			break;
 
+		case 8:
+			/* -fdump=<scope> : Add sections for dump code generation */
+			cobc_def_dump_opts (cob_optarg, 1);
+			break;
+
+		case 13:
+			/* -fno-dump=<scope> : Suppress sections in dump code generation */
+			if (cob_optarg) {
+				cobc_def_dump_opts (cob_optarg, 0);
+			} else {
+				cb_flag_dump = COB_DUMP_NONE;
+			}
+			break;
+
```

Comment:
That's not in the changelog - which commit was that in are there related changes missing?

Note: this needs more in-depth checks as the codegen for the dump was completely changed in 4.x from "static codegen of dump code" to "symbol-based dynamic dump code" (and generation of the symbols instead of the data).

Merges may get complicated here, because codegen changes would likely result in changes of the generated symbol list (or: the runtime part doing the magic).

Reply:
This is just code moved from second pass of option parsing to first pass by commit 4729.
The code was already there in trunk (see a the few next lines).

Reply:
Ah, I see - thanks for checking.